assets/src/editor/events/ticket-assignments-manager/submit-ticket-assignments-button.js

```diff
@@ -39,9 +39,7 @@ const SubmitTicketAssignmentsButton = ( {
 	} );
 	return useMemo( () => (
 		<FormSubmitButton
-			onClick={ ( click ) => {
-				processChanges( click );
-			} }
+			onClick={ () => processChanges() }
```

Comment:
This could even be `onClick={ processChanges }`

Reply:
This was done on purpose this way so as not to pass along the click event to `processChanges()`
The reason being is because `processChanges()` performs some async logic and if passed the click event, will forward that to the `closeEditor()` callback which will attempt to call `preventDefault()` on it. The problem is that the click event is a React Synthetic Event and will have its values nulled out by the time the async functions finish, so calling `preventDefault()` results in a console warning.

Reply:
Sounds great! Thank you for the clarification.